config/examples/nnf_v1alpha1_nnfcontainerprofiles.yaml

```diff
@@ -1,15 +1,33 @@
 apiVersion: nnf.cray.hpe.com/v1alpha1
 kind: NnfContainerProfile
 metadata:
-  name: randomly-fail
+  name: example-success
 data:
-  retryLimit: 6
-  # postRunTimeoutSeconds: 0
   storages:
     - name: DW_JOB_foo-local-storage
       optional: false
     - name: DW_PERSISTENT_foo-persistent-storage
+      optional: true
+  template:
+    spec:
+      containers:
+      - name: forever
```

Comment:
I've been matching the container name with the profile name

Reply:
Good call. I overlooked that.